pwndbg/commands/version.py

```diff
@@ -144,8 +146,19 @@ def bugreport(run_browser=False, use_gh=False):
 
     gdb_config = gdb.execute("show configuration", to_string=True).split("\n")
     all_info = all_versions()
+    os_info = system()
 
     current_setup = "Platform: %s\n" % platform()
+
+    if os_info.lower() == "linux" and os.path.isfile("/etc/os-release"):
+
```

Comment:
nitpick: the blank line here is unnecessary, it doesn't really help readability.